docs/dev/debugging.md

```diff
@@ -1,44 +1,67 @@
 # Debugging vs Code plugin and the Language Server
 
-**NOTE:** the information here is mostly obsolete
+## Prerequsities
 
-Install [LLDB](https://lldb.llvm.org/) and the [LLDB Extension](https://marketplace.visualstudio.com/items?itemName=vadimcn.vscode-lldb).
+- Install [LLDB](https://lldb.llvm.org/) and the [LLDB Extension](https://marketplace.visualstudio.com/items?itemName=vadimcn.vscode-lldb) VSCode extensions.
+- Open the root folder in VSCode. Here you can access the preconfigured debug setups.
 
-Checkout rust rust-analyzer and open it in vscode.
+  <img height=150px src="https://user-images.githubusercontent.com/36276403/74611090-92ec5380-5101-11ea-8a41-598f51f3f3e3.png" alt="Debug options view">
 
-```
-$ git clone https://github.com/rust-analyzer/rust-analyzer.git --depth 1
-$ cd rust-analyzer
-$ code .
-```
+- Install all TypeScript dependencies
+  ```bash
+  cd editors/code
+  npm i
```

Comment:
Maybe `npm install` for us plebs :-).

Reply:
Might you propose a change here?

Reply:
I mean "commit suggestion", though, ah, I'll do it)

Reply:
I made one, but it doesn't show up on this page.

Reply:
Saw that one, already accepted!